pkg/demoinfocs/common/player.go

```diff
@@ -454,7 +458,12 @@ func (p *Player) PositionEyes() r3.Vector {
 	}
 
 	pos := p.Position()
-	pos.Z += float64(p.Entity.PropertyValueMust("localdata.m_vecViewOffset[2]").FloatVal)
+	if p.demoInfoProvider.IsSource2() {
+		// TODO Find out where we can find the offset in Source 2 demos
+		return pos
```

Comment:
Like the velocity, I'm not sure it's available now. I didn't find a replacement.

Reply:
can you open a issue for these as a bug?

Reply:
done https://github.com/markus-wa/demoinfocs-golang/issues/406